NAMING_CONVENTIONS.md

```diff
@@ -0,0 +1,37 @@
+Arnold USD naming conventions
+=======================
+
+The Arnold USD implementation supports arnold-specific attributes through custom schemas.
```

Comment:
How about:
"In order to expose the entirety of the Arnold features in USD, we supply custom schemas for Arnold primitives in USD, and we support Arnold-specific attributes on regular USD prims. This documents the naming conventions in use to map Arnold node and parameters names in USD."